app/src/renderer/components/wallet/SendModal.vue

```diff
@@ -23,7 +23,7 @@
     </tm-form-group>
 
     <tm-form-group
-      :error="$v.fields.address.$dirty && $v.fields.address.$invalid"
+      :error="$v.fields.address.$error && $v.fields.address.$invalid"
```

Comment:
@faboweb i think this is incorrect. `$dirty` needs to be included so we don't display errors on inputs that users haven't interacted with yet.

Reply:
Maybe we can do the check on focusOut (discussion live)

Reply:
next PR

Reply:
> Maybe we can do the check on focusOut (discussion live)

is there an issue for this action item ?